README.md

```diff
@@ -216,15 +233,10 @@ The model is loaded once and processes questions sequentially. The questions rem
 ```bash
 python example.py \
   --jsonl_path examples/examples.jsonl \
-  --model_path sensenova/SenseNova-SI-InternVL3-8B 
+  --model_path sensenova/SenseNova-SI-1.1-InternVL3-8B 
 # --model_path OpenGVLab/InternVL3-8B 
 ```
 
 ### Evaluation
 
-To reproduce the benchmark results above, please refer to [EASI](https://github.com/EvolvingLMMs-Lab/EASI) to evaluate SenseNova-SI on mainstream spatial intelligence benchmarks.
-
-
-## What's next?
-
-We will release the accompanying technical report shortly. Please stay tuned!
\ No newline at end of file
+To reproduce the benchmark results above, please refer to [EASI](https://github.com/EvolvingLMMs-Lab/EASI) to evaluate SenseNova-SI on mainstream spatial intelligence benchmarks.
```

Comment:
Shall we add an citation section?

@article{sensenova-si,
  title         = {Scaling Spatial Intelligence with Multimodal Foundation Models},
  author        = {Cai, Zhongang and Wang, Ruisi and Gu, Chenyang and Pu, Fanyi and Xu, Junxiang and Wang, Yubo and Yin, Wanqi and Yang, Zhitao and Wei, Chen and Sun, Qingping and Zhou, Tongxi and Li, Jiaqi and Pang, Hui En and Qian, Oscar and Wei, Yukun and Lin, Zhiqian and Shi, Xuanke and Deng, Kewang and Han, Xiaoyang and Chen, Zukai and Fan, Xiangyu and Deng, Hanming and Lu, Lewei and Pan, Liang and Li, Bo and Liu, Ziwei and Wang, Quan and Lin, Dahua and Yang, Lei},
  journal       = {arXiv preprint arXiv:2511.13719},
  year          = {2025}
}